karrot/groups/models.py

```diff
@@ -63,8 +57,15 @@ def annotate_yesterdays_member_count(self):
         )
 
 
+class GroupManager(models.Manager):
+    def create(self, *args, **kwargs):
+        kwargs['theme'] = kwargs.get('theme', settings.GROUP_THEME_DEFAULT.value)
+        kwargs['status'] = kwargs.get('status', settings.GROUP_STATUS_DEFAULT.value)
```

Comment:
I think this can be removed, or do you need to override the default group status? I commented on this before, but I didn't understand your rationale so far.

Reply:
It would be useful to set up a playground in an automated way. Like dev.foodsaving.world there could be a dev.plantsharing.de. Don't you think it would be useful to make karrot more configureable?

However, it's not nice that ``themes`` does not fit well if the status-enum is included in there …

Reply:
(Your previous comment must have slipped by, sorry)

Reply:
> It would be useful to set up a playground in an automated way. Like dev.foodsaving.world there could be a dev.plantsharing.de.

Maybe we should talk about that in a different issue or PR, I'm not sure if this really helps in the way you intend it.

> Don't you think it would be useful to make karrot more configureable

Yes, on a case-by-case basis :)